src/fs/metadata_log.hh

```diff
@@ -201,6 +209,8 @@ private:
     void memory_only_add_dir_entry(inode_info::directory& dir, inode_t entry_inode, std::string entry_name);
     void memory_only_delete_dir_entry(inode_info::directory& dir, std::string entry_name);
 
+    void finish_writing_cluster(cluster_id_t cluster_id);
```

Comment:
Maybe `finish_writing_data_cluster` to make it more clear that it is used only with data clusters?

Reply:
It was resolved without any action or comment.

Reply:
my bad, applied now